frontend/src/app.tsx

```diff
@@ -60,16 +59,12 @@ import StudentLayer0 from './Pages/StudentLayer0.tsx';
 import StudentLayer2 from './Pages/StudentDashboard.tsx';
 import AdminLayer1 from './Pages/AdminLayer1.tsx';
 import HelpfulLinks from './Pages/HelpfulLinks.tsx';
-import { TitleManager } from './Components/TitleManager.tsx';
 
 const WithAuth: React.FC = () => {
     return (
         <AuthProvider>
             <ToastProvider>
-                <PathValueProvider>
-                    <TitleManager />
```

Comment:
TitleManager was setting the document title so that the history wasn't just UnlockEd. Instead it read "Knowledge Center - UnlockEd, Operational Insights - UnlockEd, etc."

Can we get that functionality back?

@PThorpe92